camp/leo-miniProject/src/main/java/frc/robot/commands/XBoxLeftBumper.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import frc.robot.Constants;
+import frc.robot.subsystems.TalonMotorSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** An example command that uses an example subsystem. */
+public class XBoxLeftBumper extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final TalonMotorSubsystem m_subsystem;
```

Comment:
we don't use the 'm_' naming convention that the rest of WPILIB uses. We use lowerCamelCase for variables, so this one should just be called 'subsystem'. Although a more descriptive name would be motorSubsystem.